frontend/e2e/pages/protected/apply/adult-child-page.ts

```diff
@@ -1,5 +1,39 @@
+import type { Page } from '@playwright/test';
+
 import { BasePage } from '../../base-page';
 
+interface FillOutAddressArgs {
+  address: string;
+  city: string;
+  country: string;
+  page: Page;
+  postalCode: string;
+  province: string;
+}
+
+interface fillApplicantInformationFormArgs {
+  firstName: string;
+  lastName: string;
+  sin: string;
+  day: string;
+  month: string;
+  year: string;
+  dtcEligible?: boolean;
+  page: Page;
+}
+
+interface fillChildrenInformationFormArgs {
```

Comment:
```suggestion
interface FillChildrenInformationFormArgs {
```

Same comment for other pages